array-methods-es6/index.html

```diff
@@ -0,0 +1,114 @@
+---
+feature_name: Array Methods (ES6)
+chrome_version: 45
+feature_id: 6732923508097024
+---
+<style>h2 { margin-top: 48px; }</style>
```

Comment:
Could you file a PR against the common CSS file to introduce this tweak if you feel like it significantly improves the layout? It doesn't make as much sense to be a one-off inline style.